migrator/README.md

```diff
@@ -1,23 +1,99 @@
+`Postgres` was a new potential datastore targeted by the migrator.
+
+   In 3.73, two sets of data migrations were possible: key-value store migrations, and data move migrations
+   (from `BoltDB` and `RocksDB` to `Postgres`). The migration sequence is the following: first all key-value
+   data migrations are applied, then all data move migrations are applied.
+
+    - Key-value store data migrations have the form `m_{currentDBVersion}_to_m_{currentDBVersion+1}_{summary_of_migrations}` .
+    - Data move migrations have the form `n_{postgresSchemaVersion}_to_n_{postgresSchemaVersion+1}_{moved_data_type}` .
+    
+3. After 3.73, the key-value stores are deprecated and `Postgres` becomes the only data store.  
+
+   The migration returns to the old scheme, restarting from the database version after all data moves to `Postgres`.
+
+   All migrations again have the form `m_{currentDBVersion}_to_m_{currentDBVersion+1}_{summary_of_migrations}` .
+
 ## How to write new migration script
 
 Script should correspond to single change. Script should be part of the same release as this change.
 Here are the steps to write migration script:
 
-1. Lookup current database version in `pkg/migrations/internal/seq_num.go` file
+1. Lookup the current database version (`CurrentDBVersionSeqNum`) in `pkg/migrations/internal/seq_num.go` file. 
+
+    The selected variable will be referred to as `currentDBVersion` in the next steps.
+ 
 2. Under `migrations` folder create new folder with name
 `m_{currentDBVersion}_to_m_{currentDBVersion+1}_{summary_of_migration}`
-    1. Ensure that the `summary_of_migration` follows the naming convention of previous migrations, i.e., postfix `_policy` if it modifies policies
-3. Create at least two files: `migration.go` and `migration_test.go`. These files should belong to package `m{currentDBVersion}tom{currentDBVersion+1}`
-4. To better understand how to write these two files, look at existing examples: [#1](https://github.com/stackrox/rox/pull/8609) [#2](https://github.com/stackrox/rox/pull/7581) [#3](https://github.com/stackrox/rox/pull/7921) in `migrations` directory. Avoid depending on code that might change in the future as **migration should produce consistent results**.
+
+    1. Ensure that the `summary_of_migration` follows the naming convention of previous migrations,
+        i.e., postfix `_policy` if it modifies policies
+
+3. Create at least two files: `migration.go` and `migration_test.go`. These files should belong to package
+`m{currentDBVersion}tom{currentDBVersion+1}`
+
+4. The `migration.go` file should contain at least the following elements:
+    ```go
+   import (
+       "github.com/stackrox/rox/migrator/migrations"
+       "github.com/stackrox/rox/migrator/types"
+       // If needed for the sequence number management.
+       pkgMigrations "github.com/stackrox/rox/pkg/migrations"
+   )
+   
+   var (
+       startSeqNum = {curentDBVersion}
```

Comment:
I guess you call it `startingSeqNum` in this PR so let's rename here for clarity. I also suggest to add a comment here saying how we recommend to declare the value, specifically via `pkgMigrations.BasePostgresDBVersionSeqNum() + N` and say how to figure out `N`. Maybe also clarify this relationship above around L44.